docs/pages/get-started/create-a-new-app.mdx

```diff
@@ -25,6 +27,8 @@ Before creating a new Expo app, you have to make sure that:
   cmdCopy="npx create-expo-app my-app && cd my-app"
 />
 
+> You can also use the `--template` option with the `create-expo-app` command. Run `npx create-expo-app --template` to see the list of available templates.
```

Comment:
```suggestion
> **info** You can also use the `--template` option with the `create-expo-app` command. Run `npx create-expo-app --template` to see the list of available templates.
```

Let's emphasize this message a bit more. 😉

Reply:
I think we should emphasize other callouts too in this doc and others in the "Get started" pages.

Will a create a separate PR for it.